### DISCUSSION TITLE
zh/docs/function/

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1390] [Docs comments]
# zh/docs/function/

使用内置函数或指定自定义函数

https://casbin.org/zh/docs/function/

**Comment by mdbsliguo** [comment]
How can `keyMatch` and `globMatch` be the same? What’s the difference between them?

**Reply by hsluoyz** [reply]
@mdbsliguo the latter uses glob match, see details at: https://github.com/casbin/casbin/issues/396
